spyder/plugins/completion/kite/widgets/install.py

```diff
@@ -0,0 +1,389 @@
+# -*- coding: utf-8 -*-
+
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""Kite installation widget."""
+
+# Standard library imports
+import sys
+
+# Third-party imports
+from qtpy.QtCore import QEvent, QObject, QSize, Qt, QUrl, Signal
+from qtpy.QtGui import QDesktopServices, QMovie, QPixmap
+from qtpy.QtWidgets import (QApplication, QDialog, QHBoxLayout, QMessageBox,
+                            QLabel, QProgressBar, QPushButton, QVBoxLayout,
+                            QWidget)
+
+# Local imports
+from spyder.config.base import _, get_image_path
+from spyder.config.gui import is_dark_interface
+from spyder.config.manager import CONF
+from spyder.utils import icon_manager as ima
+from spyder.plugins.completion.kite.utils.install import (ERRORED, INSTALLING,
+                                                          FINISHED, CANCELLED)
+
+
+class KiteIntegrationInfo(QWidget):
+    """Initial Widget with info about the integration with Kite."""
+    # Signal triggered for the 'Learn more' button
+    sig_learn_more_button_clicked = Signal()
+    # Signal triggered for the 'Install Kite' button
+    sig_install_button_clicked = Signal()
+    # Signal triggered for the 'Dismiss' button
+    sig_dismiss_button_clicked = Signal()
+
+    def __init__(self, parent):
+        super(KiteIntegrationInfo, self).__init__(parent)
+        # Images
+        images_layout = QHBoxLayout()
+        if is_dark_interface():
+            icon_filename = 'spyder_kite.svg'
+        else:
+            icon_filename = 'spyder_kite_dark.svg'
+        image_path = get_image_path(icon_filename)
+        image = QPixmap(image_path)
+        image_label = QLabel()
+        image_label.setPixmap(image)
+
+        images_layout.addStretch(0)
+        images_layout.addWidget(image_label)
+        images_layout.addStretch(0)
+
+        # Label
+        integration_label = QLabel(
+            _('''Now Spyder can use <a href="https://kite.com/">Kite</a> to '''
```

Comment:
There's no need to use triple quotes here. Please use double quotes and escape the one inside the string, like this:

```python
"Now Spyder can use <a href=\"https://kite.com/\">Kite</a> to "
```

You need to do the same for all strings below this one.